src/webview-scripts/GraphExport.ts

```diff
@@ -3,11 +3,11 @@
  * Licensed under the MIT License. See License.txt in the project root for license information.
  */
 
-import { IVSCode } from "./types.js";
+import { IVSCode, vscode } from "./types.js";
 
-declare const acquireVsCodeApi: () => IVSCode;
+// declare const acquireVsCodeApi: () => IVSCode;
 
-export const vscode = acquireVsCodeApi();
+// export const vscode = acquireVsCodeApi();
```

Comment:
This cannot be changed without changing all other renderers, which use this vscode export.

Better you check all the other graphs, if they still work with your patch.

Reply:
All the visualisations work, including exports.

There is an issue with "show all railway road" for parse grammars, but works fine for lexer grammars.
I don't think this is an issue introduced by this PR.

Reply:
@mike-lischke please let me know is this is a known issue with show all rule in the railroad graph.

Reply:
The following is from the webview debug tool.
This is from the GoParser.g4 grammar in https://github.com/antlr/grammars-v4.git

<img width="408" alt="image" src="https://github.com/mike-lischke/vscode-antlr4/assets/769330/c779fdad-14de-459e-b1dd-0bfc8b3e40bf">

Reply:
I don't think this error comes from my extension. Local tests show no problem there.